doc/source/install/others.rst

```diff
@@ -242,6 +242,78 @@ files::
 
 Shows all ``configure`` options.
 
+``--enable-apache-arrow``
++++++++++++++++++++++++++
+
+Enables Apache Arrow features.
```

Comment:
```suggestion
Enables Apache Arrow support. You can use not only Apache Arrow IPC streaming format output but also multithreading processing that is used in :ref:`select-n-workers` and :doc:`/reference/functions/query_parallel_or` with Apache Arrow support.
```

Oh... I don't have a document about Apache Arrow IPC stream format output in https://groonga.org/en/docs/reference/command/output_format.html ...